src/main/java/at/hannibal2/skyhanni/config/features/dev/DebugConfig.java

```diff
@@ -21,6 +21,11 @@ public class DebugConfig {
     @ConfigEditorBoolean
     public boolean commandLogs = false;
 
+    @Expose
+    @ConfigOption(name = "Always in SkyBlock", desc = "Enables SkyHanni's features everywhere, not just SkyBlock.")
```

Comment:
change the wording. since this is in dev category, you can be more technical. something like "this sets skyhannis skylbock detection to always true"

Reply:
I do not like this option as this check is here to stop crashes and other user facing errors. Being an option in the config allows any user to enable it and then they will complain. If a feature needs to work outside of skyblock we already have a place for that. Also I feel that this is so you can use the quickMaths feature outside of skyblock in something like the pit, which id consider a cheat anyways and would not approve for being in skyhanni